src/Nox.Types.EntityFramework/Types/User/UserConverter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using System.Linq.Expressions;
+
+namespace Nox.Types.EntityFramework.Types;
+
+/// <summary>
+/// The user converter.
+/// </summary>
+public class UserConverter : ValueConverter<User, string>
```

Comment:
Indeed we lose it and that was a concern initially, but since value objects are immutable and they are validated at the time when they're stored, the type really doesn't matter but again is significant from the auditing context.